src/condition/index.js

```diff
@@ -0,0 +1,20 @@
+import AllCondition from "./all-condition";
+import AnyCondition from "./any-condition";
+import ConditionConstructor, { TopLevelConditionConstructor } from "./condition-constructor";
+import neverCondition from "./never-condition";
+import ComparisonCondition from "./comparison-condition";
+import ConditionReference from "./condition-reference";
+import NotCondition from "./not-condition";
+
+export default ConditionConstructor
+
+export {
+  AllCondition,
+  AnyCondition,
+  ConditionConstructor,
+  neverCondition,
```

Comment:
nit: Why's the casing different here?

Reply:
`neverCondition` is a constant the others are classes. more a style question then anything else.